core/src/test/java/com/sequenceiq/distrox/v1/distrox/service/DistroXServiceTest.java

```diff
@@ -80,6 +86,9 @@ void setUp() {
         MockitoAnnotations.initMocks(this);
```

Comment:
This will likely not be needed after annotating the test class with `@ExtendWith(MockitoExtension.class)`.